src/events/processSignatureRequests.js

```diff
@@ -42,7 +42,10 @@ function processSignatureRequestsBuilder(config) {
           throw new Error(`RPC Connection Error: submitSignature Gas Estimate cannot be obtained.`)
         }
         logger.info(
-          { eventTransactionHash: signatureRequest.transactionHash },
+          {
+            eventTransactionHash: signatureRequest.transactionHash,
+            rpcResponse: e.message
```

Comment:
Please consider to add this changes as part of a separate PR. Since it is not related to stress testing but related to core functionality.

Reply:
Sure. I rebased the branch to remove that commit.